### DISCUSSION TITLE
CRC32 not found for NES games

### DISCUSSION THREAD

**Discussion by Swilliamson4897** [sanni/cartreader#837] [Need help]
I just completed my build yesterday 8/7/2023 (HW5 Build 12.8(Beta) and started to attempt to pull roms from my NES collection of games. The problem seems to be with the CRC32. On the initial connection it does often find the cart correctly but when reading the ines it doesn't find the crc32 which seems to constantly change at each read. I did check for bridged connections and cleaned the games some of which I have multiple copies and get different results.
[SuperMarioBros2.txt](https://github.com/sanni/cartreader/files/12292486/SuperMarioBros2.txt)
[SuperMarioBros2.txt](https://github.com/sanni/cartreader/files/12292488/SuperMarioBros2.txt)
[SuperMarioBros2.txt](https://github.com/sanni/cartreader/files/12292492/SuperMarioBros2.txt)

I have tried multiple games Dragon Warrior, Ghosts n' Goblins, Bubble Bobble, Mega Man 2, Super Mario Bros. 2, Super Mario Bros. 3. Stinger, Rygar. I am waiting for some other carts for SNES, Game Boy and Sega from some friends to see if they read correctly or not. I have read many many posts but, none seem to be a solution to the issue I am having. I have even looked at the hex data in the .nes files and the correct headers are there but are not playable in RetroArch with the Nestopia Core which is the current emulator that I am playing with at the moment.

**Comment by Ancyker** [comment]
Bad solder joints or dirty cartridges pins are the most likely culprits.
